### PR TITLE
Tarjans Algorithm C++ code

### PR DESCRIPTION
**Pull request by riddhij16**
For issue #1862 
Please review @nakul-19 @raksha009 @plazzy99 @tarun26091999 @rish-singhal

### CODE REVIEW

File: Graph/readme.md [hunk 1]
```diff
@@ -36,12 +36,13 @@ A graph is a pair (V, E), where V is a set of nodes, called vertices, and £ is
 
 ## Questions :
 
-- Breath First Search ----> [C++](/Code/C++/BFS.cpp) | [Java]() | [Python]()
+- Breath First Search ----> [C++](/Code/C++/BFS.cpp) | [Java] | [Python]()
```
Comment (raksha009) [inline Graph/readme.md:39]: Hey, I think you changed line 39 mistakenly. Pl correct ir.

File: Code/C++/Tarjans_Algo_SCC.cpp [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+Output 2:
+SCC is 4
+SCC is 3
+SCC is 1 2 0
+
+Time complexity: O(V+E)
+ 
+*/ 
+
+/*
```
Comment (raksha009) [inline Code/C++/Tarjans_Algo_SCC.cpp:137]: Remove the applications part

**Review by raksha009 — CHANGES_REQUESTED** [review]

File: Code/C++/Tarjans_Algo_SCC.cpp (hunk 2 above)
Comment (riddhij16) [reply Code/C++/Tarjans_Algo_SCC.cpp:137]: Okay sir @raksha009 
Have done the changes again.. pls have a look

File: Graph/readme.md (hunk 1 above)
Comment (riddhij16) [reply Graph/readme.md]: > Hey, I think you changed line 39 mistakenly. Pl correct ir.

done

**Comment by riddhij16** [comment]
For issue #1862
I have resolved the asked changes.. 
Please review @nakul-19 @raksha009 @plazzy99 @tarun26091999 @rish-singhal

File: Code/C++/Tarjans_Algo_SCC.cpp (hunk 2 above)
Comment (raksha009) [reply Code/C++/Tarjans_Algo_SCC.cpp:137]: Hey, the applications are still there. Plz remove them!
Also, Add space complexity and mention V and E as well

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by riddhij16** [comment]
Sorry, sir, I have done it now.. pls check @plazzy99

**Pull request merged**